tests/tests_fabric/strategies/test_xla.py

```diff
@@ -86,7 +86,7 @@ def test_tpu_reduce():
 def test_xla_mp_device_dataloader_attribute(_, monkeypatch):
     import torch_xla.distributed.parallel_loader as parallel_loader
 
-    mp_loader_mock = Mock()
+    mp_loader_mock = Mock(autospec=parallel_loader.MpDeviceLoader)
```

Comment:
@carmocca I tried spec as well in the original PR, but it's not enough because here we are attempting to mock a type. The isinstance check won't be so easy to pass `isinstance(x, mock)`. I'm not sure how to properly do this.

Reply:
Works by overriding isisntancecheck

Reply:
That's it! I completely forgot that I used the same trick in test_ddp_module_state_dict